community/index/src/test/java/org/neo4j/index/internal/gbptree/GBPTreeTest.java

```diff
@@ -436,6 +449,66 @@ public void shouldAllowClosingWriterMultipleTimes() throws Exception
         // THEN that should be OK
     }
 
+    @Test
```

Comment:
Actually, the three added tests does test both checkpoint and close, don't they?